src/picosvg/svg.py

```diff
@@ -448,6 +448,9 @@ def _stroke(self, shape):
         # make a new path that is the stroke
         stroke = shape.as_path().update_path(shape.stroke_commands(self.tolerance))
 
+        # skia stroker returns paths with 'nonzero' winding fill rule
```

Comment:
explaining why, ideally citing a link, would be nice

Reply:
this is from empirical observations, as stroked paths returned from skia often contain self-intersecting outlines (which would draw holes if interpreted as "evenodd"). We need to override the original shape's fill_rule after stroking, otherwise if the original path's fill rule (before stroking) was "evenodd" and we don't fix it to "nonzero" after the stroking, then as we later perform the remove_overlaps operation on the "evenodd" filled paths (to fix their winding), we will end up with holes, as the self-intersecting stroked outlines will be mistakenly filled as "evenodd"..

Reply:
That would make a fine comment :)